docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '2'
+services:
+  app:
+    image: friendsofsymfony/fos-http-cache
```

Comment:
If so _(no need to have image on the hub which can be confusing and become out of date)_ the `image` param can be omitted.